sse2neon.h

```diff
@@ -239,6 +239,9 @@ typedef int64x2_t __m128i; /* 128-bit vector containing integers */
 #define vreinterpretq_f64_m128d(x) (x)
 #else
 #define vreinterpretq_m128d_f32(x) (x)
+#define vreinterpretq_m128d_s64(x) vreinterpretq_f32_s64(x)
+
+#define vreinterpretq_s64_m128d(x) vreinterpretq_s64_f32(x)
```

Comment:
Put this line right after `#define vreinterpretq_f32_m128d(x) (x)`.

Reply:
why?

Reply:
Other macros which are the same class are put together.
And you have extra blank line at `245`.

Reply:
I wonder that can I remove the blank line, line 243, since I think `vreinterpretq_s64_m128d` and `vreinterpretq_m128d_s64` are the same class. However, `vreinterpretq_m128d_f32` is not in the same class of they do.

Reply:
The class is based on how the other macros do.
It does not mean which one is correct or wrong.
What I ask for is the consistency.

Reply:
Oh I get the point how do you sort each macro. I thought you sort them depends on the name of macro. However, these macros are sorted depends on the implementation instead.

Reply:
My previous comments might not consider well.
After consideration, here are two ways I suggest to do:

1. Follow the macros which are defined when A64(aarch64) is used
```c
#define vreinterpretq_m128d_s64(x) vreinterpretq_f32_s64(x)
#define vreinterpretq_m128d_f32(x) (x)

#define vreinterpretq_s64_m128d(x) vreinterpretq_s64_f32(x)
#define vreinterpretq_f32_m128d(x) (x)
```

2. Based on the classification of the other macros.
```c
#define vreinterpretq_m128d_s64(x) vreinterpretq_f32_s64(x)

#define vreinterpretq_m128d_f32(x) (x)

#define vreinterpretq_s64_m128d(x) vreinterpretq_s64_f32(x)

#define vreinterpretq_f32_m128d(x) (x)
```
However, if you change the code this way, you should change the macros like `vreinterpretq_m128d_s64` as well for consistency.

Reply:
I choose the second way